packages/devtools-launchpad/src/tools/mc/make-bundle.js

```diff
@@ -11,12 +11,11 @@ function makeBundle({ outputPath, projectPath, watch = false, updateAssets = fal
   process.env.OUTPUT_PATH = outputPath;
 
   const webpackConfig = require(path.resolve(projectPath, "webpack.config.js"));
+  if (!updateAssets) {
+    delete webpackConfig.recordsPath;
+  }
```

Comment:
got the negation wrong. If we're not updating the assets we should delete the records path...